subsys/bluetooth/mesh/blob_cli.c

```diff
@@ -542,7 +544,11 @@ void blob_cli_broadcast(struct bt_mesh_blob_cli *cli,
 
 void blob_cli_broadcast_tx_complete(struct bt_mesh_blob_cli *cli)
 {
-	k_work_submit(&cli->tx.complete);
+	if (cli->tx.ctx.next != next_chunk_delayed_send) {
```

Comment:
Sure, I can add that tomorrow morning, if no one else has opposing opinions?

Reply:
One more thing to this. In case of unicast transfer the same chunk will be sent to each target separately (`chunk_tx` will be called as many times as many targets we have and then `next` function will be called). So I guess we want to slow down this as well. This means that we also need to introduce a delay for each `chunk_tx` call. @omkar3141, am I right?

Reply:
Even if we send X chunks "at the same time" to X unicast addresses, we will still be able to open up "gaps" for other network traffic with the current change, as the added delay will be after each X chunk "batch". So a future user should be able to adjust the now added interval setting to ensure the "gaps" are big enough to not block the network (which is the main point of this change), no?

Reply:
I think in some scenarios this will still cause lags in the network especially, for example, when chunks don't fit into unsegmented message, or in certain topologies.

Reply:
By lags, do you mean that chunks might get sent later than expected or that blob transfer still occupies the network too much for other communication to pass through the network "in time"?
Because the first one I think we cannot do much about here? (as segments and how they are sent are handled in the transport layer)
And for the second one, this would "just" mean the selected interval/delay should be increased further, no?

Reply:
> By lags, do you mean that chunks might get sent later than expected or that blob transfer still occupies the network too much for other communication to pass through the network "in time"?

The second.

![delay drawio](https://github.com/user-attachments/assets/3063b8b1-1a02-480f-880a-da759d4abd25)

Reply:
I see, thanks for the diagram :) I will take a look how to implement this, meanwhile @omkar3141 could you also comment on this if this had been considered during creation/discussion of the task? And if it aligns with your "vision" for this api?